packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -329,6 +329,8 @@ export class DataStores implements IDisposable {
         pkg: Readonly<string[]>,
         isRoot: boolean,
         id = uuid()): IFluidDataStoreContextDetached {
+        assert(!id.includes("/"), "Id cannot contain slashes");
```

Comment:
This should also be added to `_createFluidDataStoreContext`

Reply:
Added. I see most paths that lead here provide a GUID for the id, and the ones that don't are marked as deprecated/to-be-removed so a test for this specific case seems unnecessary.